en-US/firefox-ios.xliff

```diff
@@ -4290,6 +4347,11 @@ Private Tab</target>
         <target>Sync</target>
         <note>This is the title for the setting that toggles Sync related notifications in the settings menu under the Notifications section.</note>
       </trans-unit>
+      <trans-unit id="Settings.Notifications.SystemNotificationsDisabledMessage.v112" xml:space="preserve">
+        <source>You turned off all %1$@ notifications. Turn them on by going to device Settings &gt; Notifications &gt; %2$@</source>
+        <target>You turned off all %1$@ notifications. Turn them on by going to device Settings &gt; Notifications &gt; %2$@</target>
+        <note>This is the footer title informing the user needs to turn on notifications in iOS Settings. The placeholder will be replaced with the app name.</note>
```

Comment:
Nit: there are two placeholders here, not one
Please explain for both in comments